README.md

```diff
@@ -27,3 +27,7 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+The invariant of this would be when n == 0 the array is 0 when n == 1 then the first 2 numbers in the array are 0 and 1.
```

Comment:
But what about the current iteration? How do you know that any iteration will give the correct result?